src/Administration/Resources/app/administration/src/module/sw-import-export/component/sw-import-export-entity-path-select/index.js

```diff
@@ -422,6 +414,37 @@ export default {
     },
 
     methods: {
+        getDefinition(isCustomField) {
+            if (isCustomField) {
+                return  {
+                    properties: this.getCustomFields(this.currentEntity || this.entityType),
+                };
+            }
+
+            // if entity is read or write protected, do not provide any property
+            const definition = Shopware.EntityDefinition.get(this.currentEntity);
+            if (definition.readProtected || definition.writeProtected) {
+                return  {
+                    properties: {},
+                };
+            }
+
+            // Filter out properties which are not AdminApiSourceAware
+            const properties = definition.filterProperties((property) => {
+                const length = property?.flags?.read_protected?.length || 0;
```

Comment:
what about the `write_protected` field flag? We should check the same for that, as I mentioned in my previous comment: https://github.com/shopware/shopware/pull/13660#discussion_r2558830474